spec/lib/retriable_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'spec_helper'
+
+RSpec.describe Retriable do
+  subject(:call) {}
+
+  describe('#retries') do
+    let(:cat) { double }
+
+    it 'makes the call' do
+      allow(cat).to receive(:maow)
+
+      RetryTest.new.retries(StandardError) { cat.maow }
+
+      expect(cat).to have_received(:maow).once
+    end
+
+    context 'when unsuccessfull calls' do
+      it 'retries multiple times' do
+        allow(cat).to receive(:maow)
+
+        expect {
+          RetryTest.new.retries(StandardError) do
+            cat.maow
+            raise('error!') # <- StandardError
+          end
+        }.to raise_error('error!')
+
+        expect(cat).to have_received(:maow).exactly(6).time
```

Comment:
These 2 expectations are a way to test the right behaviour.

Unfortunately, Rspec **has not** something like:

`has_raised('error!').exactly(6).time`

Reply:
Also, I think trying 6 times is too much,
if a call does not succeed after 2 o 3 times, it is unlikely it succeeds the 6th time.

I'd reduce it to 3 if you agree.